controllers/goharbor/harborcluster/status.go

```diff
@@ -32,7 +32,7 @@ import (
 // status is designed to track the status and conditions of the deploying Harbor cluster.
 type status struct {
 	client.Client
-	log     logr.Logger
+	log     *logr.Logger
```

Comment:
Why is it to change to a pointer

Reply:
https://github.com/go-logr/logr is upgrade. Logger change to struct from interface.